api/v1alpha3/condition_consts.go

```diff
@@ -126,4 +126,18 @@ const (
 
 	// WaitingForRemediationReason is the reason used when a machine fails a health check and remediation is needed.
 	WaitingForRemediationReason = "WaitingForRemediation"
+
+	// ExternalRemediationTemplateAvailable is set on machinehealthchecks when MachineHealthCheck controller uses external remediation.
+	// ExternalRemediationTemplateAvailable is set to false if external remediation template is not found.
+	ExternalRemediationTemplateAvailable ConditionType = "ExternalRemediationTemplateAvailable"
+
+	// ExternalRemediationTemplateNotFound is the reason used when a machine health check fails to find external remediation template.
+	ExternalRemediationTemplateNotFound = "ExternalRemediationTemplateNotFound"
+
+	// ExternalRemediationRequestAvailable is set on machinehealthchecks when MachineHealthCheck controller uses external remediation.
+	// ExternalRemediationTemplateAvailable is set to false if creating external remediation request fails.
```

Comment:
```suggestion
	// ExternalRemediationRequestAvailable is set to false if creating external remediation request fails.
```

Reply:
How badly this PR blocks [3713](https://github.com/kubernetes-sigs/cluster-api/pull/3713)? Maybe we could enhance testing in another PR?